### PR TITLE
Timeout change in the restart manager in the configuration

### PR DESCRIPTION
**Pull request by yenienserrano**
**Description:**

Added the waiting time to see if the manager is active again since it was not waiting for anything and in other ways of restarting it waits for 15000ms

**Issue:**

- #4181 

**Preconditions:**

1. Wazuh manager environments

**Test:**

1. Navigate to 'Management>Configuration'
2. Click on 'Edit configuration'
3. Change vulnerability-detector to enabled or disabled
4. Click save
5. Click restart manager button
6. Wait for it to restart

**Screenshot:**

https://user-images.githubusercontent.com/63758389/173820553-64317ce5-a3cd-413d-99b0-9b879c52c5bb.mp4

### CODE REVIEW

File: public/controllers/management/components/management/configuration/utils/wz-fetch.js [hunk 1]
```diff
@@ -280,6 +280,7 @@ export const restartNodeSelected = async (
       `Restarting ${isCluster ? selectedNode : 'Manager'}, please wait.`
     ); //FIXME: if it enables/disables cluster, this will show Manager instead node name
     isCluster ? await restartNode(selectedNode) : await restartManager();
+    await delayAsPromise(15000);
```
Comment (Desvelao) [inline public/controllers/management/components/management/configuration/utils/wz-fetch.js:283]: **question**: why this delay is required?

File: public/components/visualize/components/sample-data-warning.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const SampleDataWarning = ({ ...props }) => {
   useEffect(() => {
     (async () => {
       try {
-        const result = (await WzRequest.genericReq('GET', '/elastic/samplealerts', {})).data
+        const result = (await WzRequest.genericReq('GET', '/elastic/samplealerts')).data
```
Comment (Desvelao) [inline public/components/visualize/components/sample-data-warning.js:25]: **praise**: Good jog removing unnecessary argument.

**Review by Desvelao — APPROVED** [review]
**review**:
code :heavy_check_mark: 
test :heavy_check_mark: 

## Screenshots
image
> In the screenshot, the Wazuh API polling is visble in the API requests that are failing.

File: public/controllers/management/components/management/configuration/utils/wz-fetch.js [hunk 3]
```diff
@@ -511,8 +505,7 @@ export const checkCurrentSecurityPlatform = async () => {
   try {
     const result = await WzRequest.genericReq(
       'GET',
-      '/elastic/security/current-platform',
-      {}
```
Comment (Desvelao) [inline public/controllers/management/components/management/configuration/utils/wz-fetch.js:515]: praise: Good jog removing unnecessary argument.

File: public/controllers/management/components/management/reporting/utils/reporting-handler.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export default class ReportingHandler {
    */
   static async listReports() {
     try {
-      const result = await WzRequest.genericReq('GET', '/reports', {});
```
Comment (Desvelao) [inline public/controllers/management/components/management/reporting/utils/reporting-handler.js:22]: praise: Good jog removing unnecessary argument.

File: public/controllers/management/components/management/reporting/utils/reporting-handler.js [hunk 5]
```diff
@@ -34,8 +34,7 @@ export default class ReportingHandler {
     try {
       const result = await WzRequest.genericReq(
         'DELETE',
-        `/reports/${name}`,
-        {}
```
Comment (Desvelao) [inline public/controllers/management/components/management/reporting/utils/reporting-handler.js:38]: **praise**: Good jog removing unnecessary argument.

File: public/react-services/wz-request.ts [hunk 6]
```diff
@@ -49,7 +53,7 @@ export class WzRequest {
         headers: { ...PLUGIN_PLATFORM_REQUEST_HEADERS, 'content-type': 'application/json' },
         url: url,
         data: payload,
-        timeout: customTimeout || timeout,
```
Comment (Desvelao) [inline public/react-services/wz-request.ts:52]: **praise**: Good jog removing unused variable.

File: public/controllers/management/components/management/configuration/utils/wz-fetch.js [hunk 7]
```diff
@@ -526,24 +519,22 @@ export const checkCurrentSecurityPlatform = async () => {
...
+        ? 'Restarting cluster, it will take up to 30 seconds.'
+        : 'The manager is being restarted',
+      toastLifeTimeMs: 3000,
+    });
     isCluster ? await restartCluster() : await restartManager();
     // Dispatch a Redux action
-    updateWazuhNotReadyYet(
-      `Restarting ${isCluster ? 'Cluster' : 'Manager'}, please wait.`
-    );
-    await delayAsPromise(15000);
```
Comment (Desvelao) [inline public/controllers/management/components/management/configuration/utils/wz-fetch.js:543]: **praise**: Good jog removing unnecessary logic.

**Review by asteriscos — APPROVED** [review]
Test: :heavy_check_mark: 
CR: :heavy_check_mark: 

Peek 2022-06-17 15-56

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

**Pull request merged**